program/tests/processor.rs

```diff
@@ -27,87 +27,114 @@ use {
             set_authority, sync_native, thaw_account, transfer, transfer_checked,
             ui_amount_to_amount, AuthorityType, MAX_SIGNERS,
         },
-        processor::Processor,
         state::{Account, AccountState, Mint, Multisig},
     },
-    std::sync::{Arc, RwLock},
+    std::{
+        collections::HashMap,
+        sync::{Arc, RwLock},
+    },
 };
 
 lazy_static::lazy_static! {
     static ref EXPECTED_DATA: Arc<RwLock<Vec<u8>>> = Arc::new(RwLock::new(Vec::new()));
 }
```

Comment:
Sorry I missed this on my first pass, but all of the tests that use `set_expected_data` need to be updated.

Currently, they do a hacky thing and check the expected data during a call to `sol_set_return_data`. The correct behavior should be to remove this and instead check the return data using a `Check::return_data` on mollusk. You can do this in this PR or #19, whichever you prefer!

Reply:
Good point! I think I missed this check on #19 – I will add them there since that is the PR that adds the mollusk checks.